frontend/src/queries/nodes/InsightViz/InsightContainer.tsx

```diff
@@ -0,0 +1,252 @@
+import { Card, Col, Row } from 'antd'
+import { InsightDisplayConfig } from 'scenes/insights/InsightDisplayConfig'
+import { FunnelCanvasLabel } from 'scenes/funnels/FunnelCanvasLabel'
+import {
+    // ChartDisplayType,
+    // ExporterFormat,
+    // FunnelVizType,
+    InsightType,
+    ItemMode,
+} from '~/types'
+import { TrendInsight } from 'scenes/trends/Trends'
+import { RetentionContainer } from 'scenes/retention/RetentionContainer'
+import { Paths } from 'scenes/paths/Paths'
+import {
+    // BindLogic,
+    useValues,
+} from 'kea'
+// import { trendsLogic } from 'scenes/trends/trendsLogic'
+// import { InsightsTable } from 'scenes/insights/views/InsightsTable'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import {
+    // FunnelInvalidExclusionState,
+    // FunnelSingleStepState,
+    // InsightEmptyState,
+    InsightErrorState,
+    InsightTimeoutState,
+} from 'scenes/insights/EmptyStates'
+// import { funnelLogic } from 'scenes/funnels/funnelLogic'
+import clsx from 'clsx'
+import { PathCanvasLabel } from 'scenes/paths/PathsLabel'
+import { InsightLegend, InsightLegendButton } from 'lib/components/InsightLegend/InsightLegend'
+// import { Tooltip } from 'lib/components/Tooltip'
+// import { FunnelStepsTable } from './views/Funnels/FunnelStepsTable'
+import { Animation } from 'lib/components/Animation/Animation'
+import { AnimationType } from 'lib/animations/animations'
+// import { FunnelCorrelation } from './views/Funnels/FunnelCorrelation'
+// import { FunnelInsight } from './views/Funnels/FunnelInsight'
+// import { ExportButton } from 'lib/components/ExportButton/ExportButton'
+// import { AlertMessage } from 'lib/components/AlertMessage'
```

Comment:
normally I don't like merging in commented out code, but I get why it's here... and as long as we have a plan to get rid of it, YOLO 📈